config/locales/en.yml

```diff
@@ -880,7 +880,25 @@ en:
         action: Check here for more information
         message_html: "<strong>Your object storage is misconfigured. The privacy of your users is at risk.</strong>"
     tags:
+      moderation:
+        not_trendable: Not trendable
+        not_usable: Not usable
+        pending_review: Pending review
+        review_requested: Review requested
```

Comment:
There's a subtle difference between these two, arguably we could combine them, but I think we may want `Review requested` as a way to report tags as well in the future?

(Unless we change to polymorphic reports)